doc/source/configure.rst

```diff
@@ -47,9 +48,10 @@ on the first node::
  innodb_autoinc_lock_mode=2
 
 If you are running Red Hat or CentOS,
-add the following configuration variables to :file:`/etc/percona-xtradb-cluster.conf.d/wsrep.cnf`
+add the following configuration variables to :file:`/etc/mysql/percona-xtradb-cluster.conf.d/wsrep.cnf`
```

Comment:
For CentOS, the path should not be changed. Please revert it to /etc/percona-xtradb-cluster.conf.d/wsrep.cnf

Reply:
Done.